src/Chirp.Infrastructure/CheepRepository.cs

```diff
@@ -304,6 +304,7 @@ public List<CheepDTO> CombineCheepsAndFollowerCheeps(string AuthorName, int? pag
         //OrderByDescending doesn't sort the list but returns a new sequence, therefore we need to assign it to a new list
         var sortedCheeps = cheepsToReturn.OrderByDescending(a => a.Timestamp.Ticks).ToList();
         int? page = (pageNum - 1) * 32;
+        Console.WriteLine("So many cheeps to print wow: " + cheepsToReturn.Count  + "\n \n\n\n\n\n\n\n\n");
```

Comment:
I will quickly remove this comment and commit and push it to this pull request 😄

Reply:
Good catch! Thank you :)

Reply:
This has been resolved in commit ca7d117